Tests/Functional/Services/CurlTest.php

```diff
@@ -102,4 +102,54 @@ public function setContentType() {
         $this->assertInstanceOf('Symfony\Component\HttpFoundation\Response', $response);
         $this->assertSame('application/json', $response->headers->get('Content-Type'));
     }
+    
```

Comment:
Do you have an idea how to test both branches of the function? (with function http_parse_headers existing and without)?